rust/src/measurements/rappor/make_rappor.tex

```diff
@@ -0,0 +1,98 @@
+\documentclass{article}
+\input{../../lib.sty}
+
+
+
+
+\title{\texttt{fn make\_binary\_randomized\_response\_bool}}
+\author{Abigail Gentle}
+\begin{document}
+
+\maketitle
+
+
+
+\contrib
+
+Proves soundness of \rustdoc{measurements/fn}{make\_rappor} in \asOfCommit{mod.rs}{304ef9c2}.
+
+TODO: algorithm outline
+\section{Hoare Triple}
+\subsection{Preconditions}
+\begin{itemize}
+	\item Variable 
+\end{itemize}
+
+
+\section{Proof}
+$f/2$ refers to the probability of 
+\begin{theorem}~\cite{42852}
```

Comment:
Maybe
```suggestion
\begin{theorem}~\cite{rappor}
```
and a corresponding change below, unless the number has meaning for you.